src/module-elasticsuite-core/Model/Healthcheck/CachedHealthcheck.php

```diff
@@ -69,4 +75,27 @@ public function getDescription(): string
     {
         return $this->description;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getSeverity(): int
+    {
+        return $this->severity;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getSeverityLabel(): string
+    {
+        $severityLabels = [
+            MessageInterface::SEVERITY_CRITICAL => __('Critical'),
+            MessageInterface::SEVERITY_MAJOR    => __('Error'),
+            MessageInterface::SEVERITY_MINOR    => __('Warning'),
+            MessageInterface::SEVERITY_NOTICE   => __('Notice'),
+        ];
+
```

Comment:
Please reuse the public const of severity label once defined in the CheckInterface